src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -563,7 +563,7 @@ private static boolean isPrecededByJavadoc(DetailAST token) {
         boolean result = false;
         final DetailAST previous = token.getPreviousSibling();
         if (previous.getType() == TokenTypes.BLOCK_COMMENT_BEGIN
-                && JavadocUtils.isJavadocComment(previous)) {
+                && JavadocUtils.isJavadocComment(previous.getFirstChild().getText())) {
```

Comment:
Why did this change? How was previous method call not ok?
It is only a javadoc comment if it is in the correct position. I assume some UTs were failing?